src/registrar/config/settings.py

```diff
@@ -534,6 +534,18 @@
 SECRET_REGISTRY_KEY_PASSPHRASE = secret_registry_key_passphrase
 SECRET_REGISTRY_HOSTNAME = secret_registry_hostname
 
+# Use this variable to set the size of our connection pool in client.py
+# WARNING: Setting this value too high could cause frequent app crashes!
```

Comment:
Good call, will leave a comment.

The reason is as follows: 
Having multiple pools means having to open multiple connections concurrently (same reason why we had slow load times before). Because of that slow load time, we had to change that timeout variable on our sandboxes. That timeout period is able to handle 6 connections being created at once, but if we set the pool to like 50 or 100 that could take a while to complete. If the pool needs to restart for any reason then it'll go through that process again